src/ctl.c

```diff
@@ -362,7 +362,7 @@ static const ctl_named_node_t opt_node[] = {
 	{NAME("utrace"),	CTL(opt_utrace)},
 	{NAME("xmalloc"),	CTL(opt_xmalloc)},
 	{NAME("tcache"),	CTL(opt_tcache)},
-	{NAME("lg_tcache_max"),	CTL(opt_lg_tcache_max)},
+	{NAME("tcache_max"),	CTL(opt_tcache_max)},
```

Comment:
I think this means we need a doc change as well.

Reply:
Yeah. One thing I'm less sure is, do we want to keep `lg_tcache_max` documented? I'm leaning towards not mentioning it, but only slightly.

Reply:
Agreed. A middle ground would be to mention it, but only in the doc for tcache_max (e.g. "as a convenience, this may also be set by specifying lg_tcache_max, which will be taken to be the base-2 logartihm of the setting of tcache_max"). No strong preference either way.

Reply:
I like that better. Just updated.